eaf.el

```diff
@@ -300,6 +303,59 @@ Try not to modify this alist directly. Use `eaf-setq' to modify instead."
 Any new app should add the its name and the corresponding
 keybinding variable to this list.")
 
+
+(defvar eaf--browser-current-url nil)
```

Comment:
Change to (defvar-local eaf--bookmark-link nil)

Reply:
Not sure about this one, the variable gets before making a bookmark and there is no need to keep it buffer local because it just records the last bookmark link used.